common/OpTestBMC.py

```diff
@@ -32,8 +32,11 @@
 
 import sys
 import time
-import pxssh
 import pexpect
+try:
+    import pxssh
```

Comment:
This error, i am not seeing on my computer,it is we seeing on one of jenkins server which is having python version 2.7.5